cli/cfncluster/cfncluster.py

```diff
@@ -287,6 +292,30 @@ def poll_master_server_state(stack_name, config):
 
     return state
 
+def get_master_server_ip(stack_name, config):
+    ec2conn = boto.ec2.connect_to_region(config.region,aws_access_key_id=config.aws_access_key_id,
+                                                    aws_secret_access_key=config.aws_secret_access_key)
+
+    master_id = get_master_server_id(stack_name, config)
+
+    try:
+        instance = ec2conn.get_only_instances(instance_ids=[master_id])[0]
```

Comment:
This direct access to the first element of the list is an error condition that's worth to be checked, to avoid this kind of error to pop-up in front of the user. In this case it would mean we couldn't find the Master instance.